### ISSUE TITLE
Load a list of couress in a text file?

### ISSUE THREAD

**Issue by chaoscreater** [keethesh/UdemyCourseGrabber#10]
Hi there,

If I already have a list of URLs (let's say 200) for the free courses and they are saved into a text file, can I use that text file and load it into your app somehow? I'm just looking for a solution that can go through a predefined list of URLs and add the courses into my Udemy account.

**Comment by keethesh** [comment]
That's a good idea, I'll add that soon

**Comment by chaoscreater** [comment]
Thanks, that would be great.

**Comment by chaoscreater** [comment]
Hi, just checking the status on this one?

**Comment by keethesh** [comment]
I haven't started to add that yet, but I will definitely commit  and push tomorrow

**Comment by chaoscreater** [comment]
Hi, any updates on this yet? Sorry for pestering, but some of the free Udemy courses will expire soon...

**Comment by keethesh** [comment]
This project is for finding paid courses, ripped and shared for free on third-party websites. I'd be afraid that I can't help you here :(

**Comment by chaoscreater** [comment]
I'm confused.

Your app goes through the trouble of finding coupon codes for paid courses, while I already have a list of URLs for paid courses that are free for a limited time. Here's an example link to the paid courses that are currently free for a limited time:

https://yofreesamples.com/courses/free-discounted-udemy-courses-list/

I don't see why you can't just load a list of URLs and add the courses to your account. I mentioned this in the original post 2 weeks ago, you said you'd look into it. And after 2 weeks of constant follow up, you're not saying it's not possible. Could have just told me that at the start?

**Comment by keethesh** [comment]
My "app" DOES NOT search for coupons. It searches for pirated, aka torrent courses. For what I had told you 2 weeks ago, I had misunderstood what you meant, I though you wanted to load a list of courses to search for from a file. Moreover, I do my best to add new features to this project in my free time. Sorry for trying to help.

**Comment by chaoscreater** [comment]
Yeah, I still don't get your reasoning. 

Why go through the effort of pirating a course, when a paid course has gone free for a limited time? It's like trying to pirate a game that is currently free on Steam or the Epic Games Store. I pirate stuff too, but in this case, you're honestly just wasting time pirating instead of just adding the paid-gone-free courses to your account directly.

> 
For what I had told you 2 weeks ago, I had misunderstood what you meant

Yeah, no. You never told me anything 2 weeks ago, all you said is that what I suggested is a good idea and you'd look into it.

I get that Github projects are just projects that people contribute their free time to, that's fine and it's great that people are sharing their projects. My problem with this is that you wasted my time for 2 weeks. You could have just straight up told me that you couldn't add this feature, instead of playing with my time. That's just a dick move. 

Screenshot evidence:

![image](https://user-images.githubusercontent.com/18227319/79637891-9fab2780-81d6-11ea-9a66-289bde6eabb0.png)

**Comment by keethesh** [comment]
Let me make that clear: I though you wanted me to add a way to make the script search for a list of courses to pirate. Not to add a list of courses to your Udemy account. I understood what you meant when you sent that link. But as your idea is a good one,  I'll create another project for that, and notify you when I do. Have a good day.

**Comment by chaoscreater** [comment]
OK, that would be great. I tried to get this working, but this project has been deprecated:

https://github.com/nacimgoura/add-free-course-udemy

The description describes exactly what I'm trying to do. Maybe you can use it as a reference?

**Comment by keethesh** [comment]
See [UdemyEnrollEverywhere](https://github.com/keethesh/UdemyEnrollEverywhere)

**Comment by chaoscreater** [comment]
Thanks for the update on this one. I checked our your new project, but it doesn't seem to be the same idea that I suggested? 

My idea was that we create a .txt file with a list of URLs that already include the coupon codes for the courses. For example:

https://www.udemy.com/course/python-3-course/?couponCode=F86B6482DACBFBAE8D37
https://www.udemy.com/course/aws-technical-essentials-ultimate-training-program/?couponCode=AWS-ESSENTIALS-FREE

If you load the URLs above into your browser, you should be able to enroll in the courses straight away. Imagine if you have 200 of these URLs in a .txt file and you just need to enroll in all of those courses using a tool.

Your new project still does the same thing as your old project, it searches for coupons and adds the corresponding courses to your account. I don't want it to search for coupons. I want to make it enroll in the courses based on the list of URLs that I already have.

**Comment by keethesh** [comment]
Will add loading courses from a text file **when I have time and feel like it**. And maybe try suggesting, instead of saying "I want".

**Comment by chaoscreater** [comment]
Maybe you should learn how to read?

I've never said to you that "I want XYZ feature, go make it for me". 

In fact, I've always been saying that here's an idea of mine and here's what I want to do. I can say "I want" my idea to do whatever and then suggest this idea to others. It's not the same as saying "I want you to do something".

The whole point of my idea was to load URLs in a text file. In this thread, you mentioned on April 19th that you liked my idea and you'll create another project for it. This is what you said, I didn't force you to do anything.

Then on June 21st, you replied with a link to your new project and closed the issue. Your new project mentions this:

![image](https://user-images.githubusercontent.com/18227319/86190303-1cc02900-bb98-11ea-92a1-8d7e731b1e79.png)

Well, your new project doesn't have anything to do with the idea that it was based on. You created a new project that is exactly the same as your "UdemyCourseGrabber" project. 

Finally, if you re-read this thread, you'll understand that this isn't the first time that you've failed to read what the actual request/issue is. That's multiple fails here.

In summary:

- I've suggested an idea that is super basic and easy to understand
- You didn't understand the idea
- You later understood the idea and said you'd create a new project for it
- Your new project is irrelevant to the idea I've suggested, yet you mentioned in the description of the new project that it was based on the idea I've suggested
- You're egotistic and think that others stating the facts is a demand for you to make something happen
- I don't give a crap about this project. Go make whatever you want. You've wasted enough of my time as it is.
